.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,47 @@
+{
+	"name": "kube-dev",
+
+	// TODO publish a version of this image to the organization's package registry and then comment
+	// out the `dockerFile` configuration.
+	//"image": "ghcr.io/kube-rs/dev:v1",
+	"dockerFile": "./Dockerfile",
```

Comment:
If this is to be merged, it's probably better to publish a pre-built image based on the provided dockerfile to reduce startup time. This isn't a blocker, though.

Reply:
Yeah. With the small amount of time this takes to build locally, this isn't a problem so yet. If it becomes a heavily used feature, then we can put more work into it.